doc/API.md

```diff
@@ -176,6 +176,7 @@ axe.configure({
 	reporter: 'option',
 	checks: [Object],
 	rules: [Object],
+	standards: [Object],
```

Comment:
I know you're just following convention here, but this reads as "_an array of objects_", which I do not believe is correct.

Reply:
Ops, you're right. Shouldn't have the []